templates/data-provider/index.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useReducer } from 'react'
+import { TextField, Button } from '@oacore/design'
+import { useRouter } from 'next/router'
+
+import styles from './styles.module.css'
+import debounce from '../../utils/debounce'
+
+const initialState = {
+  helper:
+    'It can be any URL: homepage address, any data resource address, OAI-PMH endpoint or RIOXX endpoint',
+  form: {
+    dataProviderUrl: '',
+  },
+}
+
+const reducer = (state, action) => {
+  switch (action.type) {
+    case 'CLAIM_DATA_PROVIDER':
+      return state
+    case 'NEW_DATA_PROVIDER':
+      return state
+    case 'DATA_PROVIDER_NOT_FOUND':
+      return state
+    case 'CHANGE_FORM_DATA':
+      return {
+        form: {
+          ...state.form,
+          ...action.value,
+        },
+      }
+    default:
+      throw new Error()
+  }
+}
```

Comment:
We will need to rewrite the state management here. I propose to remove this as well as the empty effect below and leave this as pure as possible at the moment.